package.json

```diff
@@ -8,7 +8,7 @@
     "clean": "rm -rf static/dist",
     "nodecheckversion": "node ./scripts/node-checkversion.js",
     "preinstall": "npm run nodecheckversion",
-    "postinstall": "sh ./scripts/upstream.sh && npm run build",
```

Comment:
This would fail on Heroku where `npm install` is being executed in a `/tmp` folder which isn't a git repository.